start_esp/start_esp.py

```diff
@@ -490,6 +491,15 @@ def make_argparser():
     display this error. Setting size to 0 disables checking of client request
     body size.''')
 
+    parser.add_argument('--rewrite', action='append', help=
+    '''Internally redirect the request uri with a pair of pattern and
+    replacement. If the request uri matches perl compatible regular expression,
```

Comment:
please mention the pair is separated by space

Reply:
add 1, or 2 samples

Reply:
Updated comments